sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -180,6 +180,12 @@ object ScalaJSPlugin extends AutoPlugin {
     val fullLinkJS = TaskKey[Attributed[Report]]("fullLinkJS",
         "Link all compiled JavaScript and fully optimize", APlusTask)
 
+    val fastLinkJSOutput = TaskKey[File]("fastLinkJSOutput",
+        "Quickly link all compiled JavaScript and return the output directory")
+
+    val fullLinkJSOutput = TaskKey[File]("fullLinkJSOutput",
+        "Link all compiled JavaScript with full optimizations and return the output directory")
```

Comment:
Should these have task levels? (Or however you call the `AMinusTask` etc. thingy).

Reply:
Ah yes, indeed. I set it to `AMinusTask`.
(They're called "ranks".)